### DISCUSSION TITLE
2.6.1

### DISCUSSION THREAD

**Discussion by baimard** [baimard/gestion#410] [Announcements]
## What's Changed
* 2.6.1 by @baimard in https://github.com/baimard/gestion/pull/409

**Full Changelog**: https://github.com/baimard/gestion/compare/2.6.0...2.6.1

In the "Gestion" application for Nextcloud, the feature enabling users to share company data allows seamless collaboration between different team members. Users with access to specific company information can share files, documents, and records with others, controlling permissions such as view, edit, or delete. This ensures that sensitive company data can be securely distributed to authorized personnel while maintaining privacy and access control. The interface is designed to facilitate easy sharing and management of resources, improving teamwork and efficiency within the company.

<hr /><em>This discussion was created from the release <a href='https://github.com/baimard/gestion/releases/tag/2.6.1'>2.6.1</a>.</em>

**Comment by TheMiNuS** [comment]
Thanks a lot for the new release it enable also NC30 that's great.

Brice

**Reply by baimard** [reply]
Thank you !
